Update CrowdStrikeFalconX.py to fix pop_polling_related_args (#28919)
demisto/content: Packs/CrowdStrikeFalconX/ReleaseNotes/1_2_34.md

#### Integrations

##### CrowdStrike Falcon Intelligence Sandbox

> - Fixed an issue where the command **cs-fx-upload-file** with the arguments **submit_file=yes** and **polling=true** would fail.

Reviewer: ```suggestion
Fixed an issue where the command **cs-fx-upload-file** failed when using with **submit_file=yes** and **polling=true**.
```
